proteuslib/tools/parameter_sweep.py

```diff
@@ -15,18 +15,96 @@
 import sys
 import os
 import itertools
+import warnings
 
+from enum import IntEnum, auto
+from abc import abstractmethod, ABC 
 from idaes.core.util import get_solver
 
 # ================================================================
 
+class SamplingType(IntEnum):
+    FIXED = auto()
+    RANDOM = auto()
+
+# ================================================================
+
+class _Sample(ABC): 
+
+    def __init__(self, pyomo_object, *args, **kwargs):
+        if pyomo_object is not None:
+            # Check for indexed with single value
+            if pyomo_object.is_indexed() and len(pyomo_object.values()) == 1:
+                print(pyomo_object.values())
+                exit()
```

Comment:
```suggestion
            if pyomo_object.is_indexed() and len(pyomo_object) == 1:
```

Reply:
Whoops, looks like I left a bit of debugging in there. Thanks for cleaning it up!

Reply:
It's tedious, but we should add a simple case for the unit tests. Note something as simple as 
```
m=pyo.ConcreteModel()
m.x = pyo.Var(['a'])
```
would trigger this logic when `m.x` is passed in as the `pyomo_object`